Index: les 02/oefening 2.2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 02/oefening 2.2.py	(revision )
+++ les 02/oefening 2.2.py	(revision )
@@ -0,0 +1,7 @@
+line1='Hello Python developer...'
+line2='Welcome to the world of Python!'
+print(line1)
+print(line2)
+a=7
+b=5
+a+b
Index: les 03/3.2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 03/3.2.py	(revision )
+++ les 03/3.2.py	(revision )
@@ -0,0 +1,11 @@
+getallenlijst= [2, 4, 6, 8, 10, 9, 7]
+for getal in getallenlijst:
+    print(getal)
+
+woordenlijst= ['aap', 'boot', 'nies']
+for woorden in woordenlijst:
+    print(woorden)
+
+woord= 'voorbeeld'
+for letter in woord:
+    print(letter)
Index: les 04/4.5.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 04/4.5.py	(revision )
+++ les 04/4.5.py	(revision )
@@ -0,0 +1,6 @@
+def f(x):
+    res = x* 2 + 10
+    print(res)
+
+resultaat = f(2)
+print(resultaat)
Index: les 02/oefening 2.5.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 02/oefening 2.5.py	(revision )
+++ les 02/oefening 2.5.py	(revision )
@@ -0,0 +1,5 @@
+leeftijd1=eval(input('Geef de eerste leeftijd: '))
+leeftijd2=eval(input('Geef de tweede leeftijd: '))
+somleeftijd=leeftijd1+leeftijd2
+print('De leeftijden samen zijn ' +str(somleeftijd))
+
Index: les 03/3.6.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 03/3.6.py	(revision )
+++ les 03/3.6.py	(revision )
@@ -0,0 +1,7 @@
+getallen= [2, 4, 6, 8, 10, 9, 7]
+invoeren=eval(input('Voer een getal in: '))
+if invoeren in getallen:
+    print('Dat is correct!')
+else:
+    print('Helaas!')
+print('Tot ziens!')
Index: les 03/3.4.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 03/3.4.py	(revision )
+++ les 03/3.4.py	(revision )
@@ -0,0 +1,6 @@
+getallen= [2, 4, 6, 8, 10, 9, 7]
+aantal3= 0
+for getal in getallen:
+  if getal % 3 == 0:
+      aantal3= aantal3 + 1
+print('Het aantal getallen deelbaar door 3 is ' + str(aantal3))
Index: les 04/4.3.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 04/4.3.py	(revision )
+++ les 04/4.3.py	(revision )
@@ -0,0 +1,8 @@
+def swap (lst):
+    if len(lst) > 1:
+     lst[0], lst[1] = lst[1], lst[0]
+     return lst
+
+lst=[4,0,1,-2]
+res= swap(lst)
+print(res)
Index: les 04/4.1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 04/4.1.py	(revision )
+++ les 04/4.1.py	(revision )
@@ -0,0 +1,6 @@
+def hello (naam):
+    line= 'Welkom ' + naam + ' in de wereld van python!'
+    print(line)
+
+naam= input('Wat is je naam?')
+hello (naam)
Index: les 02/oefening 2.3.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 02/oefening 2.3.py	(revision )
+++ les 02/oefening 2.3.py	(revision )
@@ -0,0 +1,4 @@
+naam= 'Mark Rutte'
+plaats='Den Haag'
+print('Mijn naam is ' + naam + ' en ik woon in ' + plaats)
+
Index: les 02/oefening 2.6.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 02/oefening 2.6.py	(revision )
+++ les 02/oefening 2.6.py	(revision )
@@ -0,0 +1,5 @@
+vraag1=input('Wat is uw naam?: ')
+vraag2=eval(input('Wat is uw leeftijd?'))
+print(vraag1 + ', volgend jaar ben je ' + str(vraag2 +1))
+
+
Index: les 02/test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 02/test.py	(revision )
+++ les 02/test.py	(revision )
@@ -0,0 +1,86 @@
+#importing the time module
+import time
+
+#welcoming the user
+name = input("What is your name? ")
+
+print("Hello, " + name, "Time to play hangman!")
+
+print()
+
+#wait for 1 second
+time.sleep(1)
+
+print("Start guessing...")
+time.sleep(0.5)
+
+#here we set the secret
+word = "secret"
+
+#creates an variable with an empty value
+guesses = ''
+
+#determine the number of turns
+turns = 10
+
+# Create a while loop
+
+#check if the turns are more than zero
+while turns > 0:
+
+    # make a counter that starts with zero
+    failed = 0
+
+    # for every character in secret_word
+    for char in word:
+
+    # see if the character is in the players guess
+        if char in guesses:
+
+        # print then out the character
+            print(char,)
+
+        else:
+
+        # if not found, print a dash
+            print("_",)
+
+        # and increase the failed counter with one
+            failed += 1
+
+    # if failed is equal to zero
+
+    # print You Won
+    if failed == 0:
+        print("You won")
+
+    # exit the script
+        break
+
+    print()
+
+    # ask the user go guess a character
+    guess = input("guess a character:")
+
+    # set the players guess to guesses
+    guesses += guess
+
+    # if the guess is not found in the secret word
+    if guess not in word:
+
+     # turns counter decreases with 1 (now 9)
+        turns -= 1
+
+    # print wrong
+        print("Wrong")
+
+
+    # how many turns are left
+        print("You have", + turns, 'more guesses')
+
+    # if the turns are equal to zero
+        if turns == 0:
+
+        # print "You Loose"
+            print("You Loose")
+
Index: les 02/oefening 2.4.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 02/oefening 2.4.py	(revision )
+++ les 02/oefening 2.4.py	(revision )
@@ -0,0 +1,3 @@
+naam=input('Wat is je naam: ')
+plaats=input('Waar woon je: ')
+print('Mij naam is ' + naam + ' en ik woon in ' + plaats)
Index: les 03/3.7.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 03/3.7.py	(revision )
+++ les 03/3.7.py	(revision )
@@ -0,0 +1,2 @@
+for i in range(20, 61, 10):
+    print(i)
Index: les 03/3.1.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 03/3.1.py	(revision )
+++ les 03/3.1.py	(revision )
@@ -0,0 +1,6 @@
+name=input('Wat is uw naam?: ')
+leeftijd=eval(input('Wat is uw leeftijd?: '))
+if leeftijd>18:
+    print (name + ', je mag stemmen.')
+else:
+    print(name + ', je mag niet stemmen.')
Index: les 03/3.3.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 03/3.3.py	(revision )
+++ les 03/3.3.py	(revision )
@@ -0,0 +1,5 @@
+getallen= [2, 4, 6, 8, 10, 9, 7]
+som=0
+for getal in getallen:
+    som= som + getal
+print('de som van de getallen is:' + str(som))
Index: les 04/4.4.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 04/4.4.py	(revision )
+++ les 04/4.4.py	(revision )
@@ -0,0 +1,9 @@
+def f(x):
+    res = x* 2 + 10
+    return res
+
+resultaat = f(2)
+print(resultaat)
+y = 4
+resultaat1 = f(y)
+print(resultaat1)
Index: les 03/3.5.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 03/3.5.py	(revision )
+++ les 03/3.5.py	(revision )
@@ -0,0 +1,6 @@
+getallen= [2, 4, 6, 8, 10, 9, 7]
+maximum= 0
+for getal in getallen:
+    if getal > maximum:
+        maximum=getal
+print('Het grootste getal is: ' + str(maximum))
Index: les 04/4.2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- les 04/4.2.py	(revision )
+++ les 04/4.2.py	(revision )
@@ -0,0 +1,7 @@
+def getal (getallen):
+    uitkomst= max(getallen) - min(getallen)
+    print(uitkomst)
+    return (uitkomst)
+
+getallen=([4,0,1,-2])
+getal (getallen)
